src/components/Text/src/Text.vue

```diff
@@ -81,6 +81,7 @@ export default {
 }
 
 .Paragraph {
+	color: var(--color-800);
```

Comment:
Does this need a fallback? like `var(--color-800, inherit)`?

Reply:
This will have the same issue as heading where a profile can have its unique explicit value for the text color.